src/app/active-mode/Views/current-activity/VendorBounties.tsx

```diff
@@ -41,6 +40,28 @@ function mapStateToProps(state: RootState): StoreProps {
 
 type Props = ProvidedProps & StoreProps;
 
+export type DefType = 'ActivityMode' | 'Destination' | 'DamageType' | 'ItemCategory' | 'KillType';
+
+function bountiesForActivity(bounties: VendorItem[], activity: DestinyActivityDefinition) {
+  return bounties.filter(({ item }) => {
+    const info = item?.hash && pursuitsInfo[item?.hash];
+    if (!info) {
+      return false;
+    }
+
+    for (const key in info) {
+      switch (key) {
+        case 'Destination':
+          return info[key].includes(activity.placeHash);
+        case 'ActivityMode':
+          return activity.activityModeHashes.some((modeHash) => info[key].includes(modeHash));
+        default:
+          return true; //!info['ActivityMode']; // Filter out things like 'Cast your Super' #171761468
```

Comment:
I need to double check with some bounties how this actually works. May need to uncomment this line.

Reply:
Yeah, those are things like KillType and ItemCategory, which shouldn't matter one way or the other.

Reply:
I just tested it out and my crucible bounty "cast super twice" wasn't marking as completed when I was on the cosmodrome, so I uncommented the line. 

Iif you're in an activity it should look for bounties that match that activity, and then exclude for bounties that match for example (if you're in a different activity, even though the kill type matches):
```
bounty: {
   killType: Cast Super
   activityMode: Crucible  
}
```